pkg/zone/zone.go

```diff
@@ -0,0 +1,63 @@
+package zone
+
+import "github.com/cilium/cilium/pkg/lock"
+
+var zoneIDAlloc = NewZoneAllocator()
+
+// TODO(AwesomePatrol): Replace allocator with a configuration option.
```

Comment:
Don't leave TODOs in the code, but rather create a github issue that details why this is needed to be done in the future.

Reply:
This implementation was replaced with `FixedZoneMapping` added in the later commit. Would you prefer me to squash them?